### DISCUSSION TITLE
cant open start_ofd

### DISCUSSION THREAD

**Discussion by HusseinElnadi** [DIGITALCRIMINALS/OnlyFans#445] [Q&A]
Hello all
I can't open start_ofd I did everything to open it but it closes fast.
onlyfans7.6.1
Python 3.10.4
![image](https://user-images.githubusercontent.com/104869230/166578636-86a29930-31e4-4eff-b8ac-e267ef9ca310.png)
what is the problem?

**Comment by pirunthan7** [comment]
same thing onlyfans7.6.1 not opening but with command  poetry run python ./start_ofd.py it's running but not downloading anything , giving author error.  regular version doesn't have x_bc":"x-bc_value in the auth.json file. please help to fix it.

**Comment by Dragoviano** [comment]
Use this command when in OnlyFans-7.6.1 folder: pip install --upgrade --user -r requirements.txt

Now it should open.
